src/vs/workbench/contrib/terminal/browser/terminalActions.ts

```diff
@@ -1721,7 +1721,8 @@ interface IRemoteTerminalPick extends IQuickPickItem {
 
 function getSelectedInstances(accessor: ServicesAccessor): ITerminalInstance[] | undefined {
 	const listService = accessor.get(IListService);
-	if (!listService.lastFocusedList?.getSelection()?.length) {
+	const terminalService = accessor.get(ITerminalService);
+	if (!listService.lastFocusedList?.getSelection()) {
```

Comment:
We want to allow clicking off of the tabs view and right clicking on an instance. Thus, a selection length of 0 shld be allowed